common/styles/components/_event_schedule.scss

```diff
@@ -1,5 +1,13 @@
 .event-schedule {
   padding-top: 10vw;
+
+  @include respond-to('large') {
+    padding-top: 8vw;
+  }
+
+  @include respond-to('xlarge') {
+    padding-top: 18 * $spacing-unit;
```

Comment:
Not able to do this in the HTML?

Reply:
We only have helper classes that go up to 10 * $spacing-unit